cfgov/unprocessed/css/molecules/pagination.less

```diff
@@ -0,0 +1,139 @@
+/* topdoc
+  name: Pagination
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+          <nav class="m-pagination">
+              <a class="btn btn__super m-pagination_btn_prev" href="#o-pagination-content_list">
+                  <span class="btn_icon__left cf-icon cf-icon-left"></span>
+                  Previous
+              </a>
+              <a class="btn btn__super m-pagination_btn_next" href="#o-pagination-content_list">
+                  Next
+                  <span class="btn_icon__right cf-icon cf-icon-right"></span>
+              </a>
+              <form data-js-hook="validate_pagination" action="#o-pagination-content_list">
+                  <label for="m-pagination_current-page">
+                      Page
+                      <span class="u-visually-hidden">
+                          number out of {{ value.pages }} total pages
+                      </span>
+                  </label>
+                  <input class="m-pagination_current-page"
+                         id="pagination_current-page"
+                         name="pagination_current-page"
+                         type="number" min="1" max="149"
+                         value="149">
+                  <span class="m-pagination_label">
+                      <span aria-hidden="true">
+                          of 149
+                      </span>
+                  </span>
+                  <button id="m-pagination_submit"
+                          type="submit">
+                      Go
+                  </button>
+              </form>
+          </nav>
+  tags:
+    - cfgov-molecules
+*/
+
+@pagination-bg-color: #F0F1F1;
+@pagination-text-color: #75787B;
+@pagination-font-size__px: 16px;
+@pagination-font-size__em: unit( @pagination-font-size__px / @base-font-size-px, em );
+@pagination-height__px: 42px;
+
+.m-pagination {
+    display: block;
+    position: relative;
+    height: @pagination-height__px;
+
+    label, &_label {
+        display: inline-block;
+        margin: 0;
+       .webfont-regular();
+        font-size: @pagination-font-size__em;
+        vertical-align: middle;
+    }
+
+    input[type="number"] {
+      .u-inline-block();
```

Comment:
Mix of use of `.u-inline-block()` and `inline-block` in the file. We could probably move away from `u-inline-block()` as it's only needed for IE6 I think.